corehq/apps/analytics/tasks.py

```diff
@@ -361,6 +375,8 @@ def track_periodic_data():
                 max_workers = domains_to_mobile_users[domain]
 
         project_spaces_created = ", ".join(get_domains_created_by_user(email))
+        if max_forms > DATADOG_USER_MAX_FORMS_LIMIT:
+            number_of_users_exceeding_max_forms += 1
```

Comment:
@snopoke So we are using `commcare.hubspot.domains_with_forms_gt_threshold: domains_with_forms_gt_threshold` but its expected to be reveal users with any domain exceeding 300 forms. Does that sounds alright?